nginx/nginx.go

```diff
@@ -85,13 +86,19 @@ func (n *nginxUpdater) signalIfRequired() {
 // Nginx implementation
 type nginxUpdater struct {
 	Conf
-	running              util.SafeBool
-	lastErr              util.SafeError
-	metricsUnhealthy     util.SafeBool
-	initialUpdateApplied util.SafeBool
-	doneCh               chan struct{}
-	nginx                *nginx
-	updateRequired       util.SafeBool
+	running                util.SafeBool
+	lastErr                util.SafeError
+	metricsUnhealthy       util.SafeBool
+	initialUpdateApplied   initialUpdateApplied
+	initialUpdateAttempted util.SafeBool
+	doneCh                 chan struct{}
+	nginx                  *nginx
+	updateRequired         util.SafeBool
+}
+
+type initialUpdateApplied struct {
```

Comment:
Can we rename this to `nginxStarted` or something similar. With `initialUpdateAttempted` we can be more specific with this one.